fMRI_Tutorial/DataCheck.ipynb

```diff
@@ -0,0 +1,473 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Notebook on how to check your fMRI data\n",
+    "\n",
+    "Authors: Eduarda Centeno\n",
+    "    \n",
+    "The goal is to show how to work with matrices, how to plot and what to check.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Let's start with the necessary packages for the following computations:\n",
+    "\n",
+    "    Requirements: \n",
+    "    \n",
+    "    - Python: 3.7.3\n",
+    "    - Numpy: 1.16.3\n",
+    "    - Matplotlib: 3.2.1\n",
+    "    - Seaborn: 0.10.1\n",
+    "    - Pandas: 1.0.3\n",
+    "    - JupyterLab: 1.2.0\n",
+    "    \n",
+    "    \n",
+    "    \n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "## Libraries\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import seaborn as sns\n",
+    "import pandas as pd\n",
+    "import glob"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Some key tips:\n",
+    "\n",
+    "    - You should check if there are NaNs, zeros, infs in the matrix.\n",
```

Comment:
Does this imply that people need to do that separately? Or would it make sense to just add that to this datacheck?

Reply:
I'll add a check for that!